tests/features/joinup_discussion/discussion.subscribe.feature

```diff
@@ -0,0 +1,32 @@
+@api
+Feature: Subscribing to discussions
+  As a member of a collection
+  I want to subscribe to interesting discussions
+  So that I can stay up to date with its evolvements.
```

Comment:
@brummbar points out that this is not limited to members of a collection. Any authenticated user can subscribe to a discussion. We need to find a different persona.